### DISCUSSION TITLE
[Feature] [ci] Add rerun bot

### DISCUSSION THREAD

**Discussion by xleoken** [apache/seatunnel#1247] [Ideas]
### Search before asking

- [X] I had searched in the [feature](https://github.com/apache/incubator-seatunnel/issues?q=is%3Aissue+label%3A%22Feature%22) and found no similar feature requirement.

### Description

Add a rerun bot which can help developers rerun the actions by themself.

### Are you willing to submit a PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by CalvinKirs** [comment]
It would be nice to be able to describe in detail how to do this.

**Comment by xleoken** [comment]
hi @CalvinKirs, I am also learning 😃
